compiler/cpp/src/main.cpp

```diff
@@ -28,6 +31,10 @@ int main(int argc, char** argv) {
     // Print output
     std::cout << output;
 
+    //auto t1 = std::chrono::steady_clock::now();
```

Comment:
still need this?

Reply:
Yes. I decided to leave this in there, so that we can investigate performance issues better in the future.